utils/docker/Dockerfile.centos.8

```diff
@@ -307,6 +307,10 @@ RUN echo -e "<settings>\n\
 
 ARG DAOS_JAVA_BUILD=$DAOS_BUILD
 
+RUN CHOICE=$(echo "" | sudo alternatives --config java | grep "java-1.8" | tr -cd '[[:digit:]]') && \
+        echo ${CHOICE:0:1} | sudo alternatives --config java
+RUN java -version
+
```

Comment:
Should this still be here for CentOS 8?

You should remove the BASE_DISTRO arg from the top of this file however as it's no longer used.